Libplanet/Assets/FungibleAssetValue.cs

```diff
@@ -440,6 +450,16 @@ public static FungibleAssetValue Parse(Currency currency, string value)
             return new FungibleAssetValue(currency, sign, major, minor);
         }
 
+        /// <summary>
+        /// Restores a value from a <paramref name="rawValue"/> of a <paramref name="currency"/>.
+        /// </summary>
+        /// <param name="currency">The currency to create a value.</param>
+        /// <param name="rawValue">The <see cref="RawValue"/> to restore.</param>
+        /// <returns>The value restored value the given <paramref name="rawValue"/>.</returns>
```

Comment:
Maybe "The value resored from"?

Reply:
Though I intended “restore﻿·d,” is it unnatural?

Reply:
I thought "value" used twice... was it intended?

Reply:
Not intnded… 🙄 I am going to fix it. Thanks!

Reply:
@limebell Amended!